<launch>
  <include file="$(find leo_erc_gazebo)/launch/leo_marsyard2021.launch"/>
  <include file="$(find leo_erc_viz)/launch/rviz.launch"/>
  <param name="/use_sim_time" value="true"/>
  <node pkg="pointcloud_process" type="localizer" name="localizer"/>
  <node pkg="pointcloud_process" type="to_traverse" name="to_traverse"/>
  <include file="$(find pointcloud_process)/launch/costmap.launch"/>
</launch>
